<?xml version="1.0"?>
<launch>

    <param name="print_interval" value= "5" />

    <include file="$(find assignment_2_2022)/launch/assignment1.launch" />

    <node pkg="rt1_2nd_assignment" type="NodeA.py" name="NodeA" launch-prefix="gnome-terminal --command" />
    <node pkg="rt1_2nd_assignment" type="NodeB.py" name="NodeB" launch-prefix="gnome-terminal --command" />
    <node pkg="rt1_2nd_assignment" type="NodeC.py" name="NodeC" launch-prefix="gnome-terminal --command" />   
</launch>
